aviso cambios en merchant order atributo email del collector (#779)
mercadopago/devsite-docs: guides/changelog/current-year.en.md

---

> ## Julio 2020

Reviewer: ```suggestion
## July 2020
```

### 08 de julio

> CHANGELOG
>

> > Cambios en la API de órdenes

Reviewer: ```suggestion
> API Orders changes
```

>
> FEATURE: FEATURE
>

> > PRODUCT: ORDENES

Reviewer: ```suggestion
> PRODUCT: ORDERS
```

> Para adecuarnos a nuevas medidas de seguridad en el manejo de datos de nuestros usuarios, vamos a dejar de mostrar el email del collector en la creación de órdenes desde el miércoles 08/07. Entendemos que este cambio no debería generar afectación ya que es el mismo email del usuario que se encuentra creando la orden.

Reviewer: ```suggestion
We update our data protection protocols to meet the highest security standards. After July 8th, we will stop displaying the seller's e-mail when creating orders. The change shouldn't cause any problems in the integration, since it's information of the person who is creating the order.
```

> You can see all the information in the [API Reference](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/reference/merchant_orders/resource/)

Reviewer: ```suggestion
You can see all the information in the [API Reference](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/reference/merchant_orders/resource/).
```

> [Ir a documentación de Órdenes](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/es/reference/merchant_orders/resource/)

Reviewer: ```suggestion
You can see all the information in the [API Reference](https://www.mercadopago[FAKER][URL][DOMAIN]/developers/en/reference/merchant_orders/resource/)
```
